src/west/app/project.py

```diff
@@ -164,9 +164,11 @@ def do_add_parser(self, parser_adder):
         parser.add_argument('--mf', '--manifest-file', dest='manifest_file',
                             help='manifest file name to use')
         parser.add_argument('-l', '--local', action='store_true',
-                            help='''use an existing local manifest repository
-                            instead of cloning one; cannot be combined with
-                            -m or --mr.''')
+                            help='''use "directory" as an existing local
+                            manifest repository instead of cloning one from
+                            MANIFEST_URL; .west is created next to "directory"
+                            in this case, and manifest.path is set to
+                            "directory"''')
```

Comment:
> and manifest.path is set to "directory"

Not completely true.
If I do `west init -l /very/log/path/manifest-repo`, then `manifest.path` will be set to `manifest-repo`, and not `/very/log/path/manifest-repo`.

Reply:
> Not completely true.

How about now? I'm using vaguer language.